app/javascript/components/student/editor/ExercismMonacoEditor.tsx

```diff
@@ -88,6 +87,27 @@ export function ExercismMonacoEditor({
         connection.onClose(() => disposable.dispose())
       },
     })
+  }, [document, listen])
+
+  useEffect(() => {
+    window.addEventListener('load', startLanguageClient)
```

Comment:
Won't work with Turbolinks. What's the goal here? It seems that you'll always want to start it if the component is active? If so, do you need an event? I'm not asking these questions so you will change it, but rather trying to figure out your thinking so we can figure out what's best together 🔢

Reply:
Your approach is a lot better and it works. I did this as I was concerned that the meta tag wouldn't be loaded by the time the React component is loaded, but didn't need to worry about that.